GenderPayGap.Core/Classes/Logger/CustomLogger.cs

```diff
@@ -32,7 +33,16 @@ public static void Fatal(string message, object values = null)
 
         private static string GetLogMessage(string message, object values = null)
         {
-            return values == null ? message : message + ". Log: {@Values}";
+            try
+            {
+                JsonConvert.SerializeObject(values);

```

Comment:
The logger doesn't use `JsonConvert` but our assumption is that it is doing something similar. This try/catch will stop us from getting caught out in future.

Reply:
Can you add a comment explaining this.
I'm worried a developer in the future will think they can remove this code because we're not using the value returned from `JsonConvert.SerializeObject(values)`